apps/framework-cli/src/infrastructure/kafka_clickhouse_sync.rs

```diff
@@ -580,29 +568,27 @@ fn map_json_value_to_clickhouse_value(
 
             if let Some(obj) = value.as_object() {
                 // Needs a null if the column isn't present
-                let column_values = inner_nested
-                    .columns
-                    .iter()
-                    .map(|col| {
-                        let col_name = &col.name;
-                        let val = obj.get(col_name);
-                        match val {
-                            Some(val) => (
-                                std_column_to_clickhouse_column(col.clone()).unwrap(),
+                let mut values: Vec<ClickHouseValue> = Vec::new();
+                let _ = inner_nested.columns.iter().map(|col| {
+                    let col_name = &col.name;
+                    let val = obj.get(col_name);
+                    match val {
+                        Some(val) => (
+                            //std_column_to_clickhouse_column(col.clone()).unwrap(),
+                            values.push(
                                 map_json_value_to_clickhouse_value(&col.data_type, val).unwrap(),
```

Comment:
We need to do something better than `unwrap` in this context. If there is an error we probably should log it and ignore. 

But `unwrap` will panic if the value is not the one intended. If this panics, the sync will stop and we don't want that.

It looks like we had some unwrap in the code before so this will make it more solid